draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1580,7 +1661,7 @@ The following Disclosures are created by the Issuer:
 
 {{examples/jsonld/disclosures.md}}
 
-The following is how a presentation of the SD-JWT with Key Binding JWT that discloses only `type`, `medicinalProductName`, `atcCode` of the vaccine, `type` of the `recipient`, `type`, `order` and `dateOfVaccination` could look like:
+The following is how an SD-JWT-KB over an SD-JWT that discloses only `type`, `medicinalProductName`, `atcCode` of the vaccine, `type` of the `recipient`, `type`, `order` and `dateOfVaccination` could look like:
 
 <{{examples/jsonld/sd_jwt_presentation.txt}}
 
```

Comment:
somewhere down below there should be a document history bullet added for this

Reply:
thanks @bifurcation - I now see a document history bullet somewhere down below for this change